File: Android/Java/UD3/Practica/app/src/main/java/com/pmdm/practica/Actividad2.java
package com.pmdm.practica;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import android.view.View;
import android.widget.Button;

public class Actividad2 extends AppCompatActivity {

    private Button boton1;

    private Button boton3;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.actividad2);
        boton1 = findViewById(R.id.Button01);
        boton3 = findViewById(R.id.Button02);

        boton1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                irAActividad1(null);
            }
        });

        boton3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                irAActividad3(null);
            }
        });
    }

    private void irAActividad1(View view){
        Intent i = new Intent(this, MainActivity.class);
        startActivity(i);
    }

    private void irAActividad3(View view){
        Intent i = new Intent(this, Actividad3.class);
        startActivity(i);
    }
}

File: Android/Kotlin/UD3/Practica/app/src/main/java/com/pmdm/practica/Actividad2.kt
package com.pmdm.practica

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class Actividad2 : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.actividad2)
        val boton1: Button = findViewById(R.id.Button01)
        val boton3: Button = findViewById(R.id.Button02)

        boton1.setOnClickListener{
            lanzarAActividad1(null)
        }

        boton3.setOnClickListener{
            lanzarAActividad3(null)
        }
    }

    private fun lanzarAActividad1(view: View?){
        val i : Intent = Intent(this, MainActivity::class.java)
        startActivity(i)
    }

    private fun lanzarAActividad3(view: View?){
        val i : Intent = Intent(this, Actividad3::class.java)
        startActivity(i)
    }
}